Zend/zend_objects_API.h

```diff
@@ -137,5 +137,20 @@ static inline zend_property_info *zend_get_typed_property_info_for_slot(zend_obj
 	return NULL;
 }
 
+static inline bool zend_check_method_accessible(const zend_function *fn, const zend_class_entry *scope)
+{
+	if (
+		!(fn->common.fn_flags & ZEND_ACC_PUBLIC)
```

Comment:
I'd prefer making this an assert, as many of the call-sites have already accounted for this. Some of them are quite hot too, and there's no guarantee the compiler will omit the repeated check.

Reply:
Symfony instruction count went up by 0.01%. Seems to confirm this is not omitted, or the function isn't inlined.